### ISSUE TITLE
TUITextView not respond to Input methods.

### ISSUE THREAD

**Issue by Naituw** [twitter-archive/twui#71]
I'm building a app totally based on TwUI, But I found that TUITextView doesn't respond to Input methods (such like Chinese Pinyin).When I type in it, the Input method's word select window doesn't show.  there has a same problem in the Twitter for Mac's search field (new tweet window works fine.)

I'v tried using a TUIViewNSViewContainer to add a NSTextView in a TUIView. but app receive a  EXC BAD ACCESS when I add the nsview container view, here is the code :

``` Objective-C
TUIView * root = [[TUIView alloc] init];
    root.backgroundColor = [TUIColor colorWithWhite:0.9 alpha:1.0];
    tuiContainer.rootView = root;
    NSTextView * textView = [[NSTextView alloc] initWithFrame:root.bounds];
    TUIViewNSViewContainer * nsViewCon = [[TUIViewNSViewContainer alloc] initWithNSView:textView];
    [root addSubview:nsViewCon];
```

Does there have any solution to handle input method using Twui? Thanks~

**Comment by lorenbrichter** [comment]
TUIViewNSViewContainer is still at around 0% complete, and ideally it's a hack that should never be necessary.  The _proper_ way to go is to make TUITextEditor and companion classes support these input methods.  This may be hard.

Easier, but more limiting: host a NSTextView outside of the TUIView hierarchy – this may require some UI changes.

**Comment by Naituw** [comment]
Thanks ~ I add NSTextView to the root TUINSView, and keep it on top, seems work!~~ I will try to implement in this way

**Comment by joshaber** [comment]
This is mostly resolved in #107.
